### PR TITLE
Created a new service and override CC thumbnail path service

### LINKED ISSUE #1369: create icons that differentiate works from files
_From @jcoyne on July 16, 2015 16:34_

Update the `SufiaHelperBehavior#work_thumbnail` with a better icon. 
https://github.com/projecthydra-labs/sufia-core/blob/master/app/helpers/sufia/sufia_helper_behavior.rb#L202

Refs projecthydra-labs/sufia-core#46

_Copied from original issue: projecthydra-labs/sufia-core#50_

### PR DESCRIPTION
**Pull request by mtribone**
Fixes #1369 

In order to distinguish between works and files, the default image for works needed to be overridden from Curation Concerns. Created work thumbnail path which inherits from CC ThumbnailPathService and new worker indexer to override the default image file 

Changes proposed in this pull request:
* Added new images for works and files from Font Awesome icons
* Created work thumbnail service in Sufia that inherits from CC and changes default image
* Overide CC work_indexer

@projecthydra/sufia-code-reviewers

### CODE REVIEW

**Comment by mtribone** [comment]
First two works use new code and image. Third work uses old default.


screen shot 2016-05-02 at 1 00 17 pm

File: app/indexers/curation_concerns/work_indexer.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module CurationConcerns
+  class WorkIndexer < ActiveFedora::IndexingService
+    include IndexesThumbnails
+    self.thumbnail_path_service = Sufia::WorkThumbnailPathService
+    def generate_solr_document
```
Comment (jcoyne) [inline app/indexers/curation_concerns/work_indexer.rb]: why is this code needed?

Comment (jcoyne) [inline app/indexers/curation_concerns/work_indexer.rb]: That is, why don't we create a Sufia::WorkIndexer that inherits from CurationConcerns::WorkIndexer?

**Comment by mtribone** [comment]
@jcoyne Used the new work indexer and added the new work thumbnail service. Give it a look when the lights turn green.

**Comment by jcoyne** [comment]
👍

**Pull request merged**